src/boundary/Boundary.h

```diff
@@ -15,31 +15,31 @@
 class Boundary {
  public:
     ~Boundary();
-    Boundary(Obstacle** obstacleList, size_t numberOfObstacles, size_t size_obstacles, size_t level = 0);
+    Boundary(Obstacle** obstacle_list, size_t number_of_obstacles, size_t size_obstacles, size_t level = 0);
     explicit Boundary(size_t level = 0);
     void init(size_t size_obstacles);
 
-    size_t* getBoundaryList() {return m_boundaryList;}
-    size_t* getBoundaryFront() {return m_boundaryFront;}
-    size_t* getBoundaryBack() {return m_boundaryBack;}
-    size_t* getBoundaryTop() {return m_boundaryTop;}
-    size_t* getBoundaryBottom() {return m_boundaryBottom;}
-    size_t* getBoundaryLeft() {return m_boundaryLeft;}
-    size_t* getBoundaryRight() { return m_boundaryRight;}
-
-    size_t* getInnerList() { return m_innerList;}
-    size_t getSize_innerList() { return m_size_innerList; }
-
-    size_t getSize_boundaryList() {return m_size_boundaryList;}
-    size_t getSize_boundaryFront() {return  m_size_boundaryFront;}
-    size_t getSize_boundaryBack() {return   m_size_boundaryBack;}
-    size_t getSize_boundaryTop() {return    m_size_boundaryTop;}
-    size_t getSize_boundaryBottom() {return m_size_boundaryBottom;}
-    size_t getSize_boundaryLeft() {return   m_size_boundaryLeft;}
-    size_t getSize_boundaryRight() { return m_size_boundaryRight;}
-
-    void updateLists(Obstacle** obstacleList, size_t numberOfObstacles, size_t size_obstacles);
-    void updateLists();
+    size_t* get_boundary_list() const {return m_boundary_list;}
+    size_t* get_boundary_front() const {return m_boundary_front;}
+    size_t* get_boundary_back() const {return m_boundary_back;}
+    size_t* get_boundary_top() const {return m_boundary_top;}
+    size_t* get_boundary_bottom() const {return m_boundary_bottom;}
+    size_t* get_boundary_left() const {return m_boundary_left;}
+    size_t* get_boundary_right() const { return m_boundary_right;}
+
+    size_t* get_inner_list() { return m_inner_list;}
```

Comment:
can be const

Reply:
71d6ea6acf5bf89eeebaa98ea5d1878cfbd1695d